persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/TestEntity.scala

```diff
@@ -43,9 +43,9 @@ object TestEntity {
     override def aggregateTag: AggregateEventShards[Evt] = Evt.aggregateEventShards
   }
 
-  final case class Appended(element: String) extends Evt
+  final case class Appended(entityId: String, element: String) extends Evt
```

Comment:
I had to add this when doing a 1:1 port of the javadsl since the javadsl tests relied on it.